apps/web/components/new-experiment/new-experiment-protocols-card.tsx

```diff
@@ -30,7 +30,15 @@ export function NewExperimentProtocolsCard({ form }: NewExperimentProtocolsCardP
   const watchedProtocols = form.watch("protocols");
   const protocols = useMemo(() => watchedProtocols ?? [], [watchedProtocols]);
 
-  // Exclude protocols already in the current protocols list
+  // Rehydrate UI from form state whenever protocols change
+  useEffect(() => {
```

Comment:
Since we already have the flow, maybe we can obliterate the protocol selection from the form and api @petar-INFO what do you think?